menu/cakemenu.html

```diff
@@ -39,111 +35,68 @@
 											<h1>Cake Menu</h1>
 										</header>
 										<p>Aenean ornare velit lacus, ac varius enim ullamcorper eu. Proin aliquam facilisis ante interdum congue. Integer mollis, nisl amet convallis, porttitor magna ullamcorper, amet egestas mauris. Ut magna finibus nisi nec lacinia. Nam maximus erat id euismod egestas. Pellentesque sapien ac quam. Lorem ipsum dolor sit nullam.</p>
-										<ul class="actions">
-											<li><a href="../orders.html" class="button big">Order Now</a></li>
-										</ul>
 									</div>
 									<span class="image object">
-										<img src="../images/bio_pic.jpg" alt="Amanda next to two-tiered wedding cake" />
+										<img src="../images/pic01.jpg" alt="placeholder" />
```

Comment:
the more i look at this page, the more i wonder if we even need this image? I am trying to think about what image would go here that wouldn't just show up again below in the product selection.

Reply:
that makes sense, I am not sure we need these images. I will remove them for now.